dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -712,6 +712,10 @@ error.invalidPhaseName=CWWKE0954E: The "{0}" checkpoint phase specified is empty
 error.invalidPhaseName.explanation=An unknown phase was specified as the value of the checkpoint option.
 error.invalidPhaseName.useraction=Review the documentation to determine valid phase names. Reissue the command using a valid checkpoint phase.
 
+error.set.securitymanager.jdk18=CWWKE0955E: The property "websphere.java.security" was set in the bootstrap.properties file while using Java "{0}". This property enables the Java Security Manager and is no longer valid when using Java 18 or later.
+error.set.securitymanager.jdk18.explanation= As part of the deprecation of the Java Security Manager, as specified by JEP 411, Liberty no longer supports enabling the Java Security Manager when using Java 18 or later.
```

Comment:
suggested edit

As part of the deprecation of the Java Security Manager, as specified by JEP 411, Liberty does not support the Java Security Manager with Java 18 or later.